Packs/CommonScripts/Scripts/GetIndicatorDBotScore/GetIndicatorDBotScore.py

```diff
@@ -26,12 +26,15 @@ def get_dbot_score_data(indicator, indicator_type, source, score):
 def iterate_indicator_entry(indicator, entry):
     indicator_type = entry["indicator_type"]
     indicator_type = INDICATOR_TYPES.get(indicator_type, indicator_type).lower()
-    sources = entry.get('sourceBrands', [])
-    sources = sources if sources else [None]
-    for source in sources:
+    sources = entry.get('moduleToFeedMap', {})
+    if entry.get('manualScore'):
+        sources[entry.get('setBy')] = {}
+    elif not sources:
+        sources[None] = {}
```

Comment:
why? isn't that creates a dummy indicator?

Reply:
@yaakovi no.
Because we have the indicator in the query and the score in the entry (here it’s just a part of the entry).
you can an example for this in https://github.com/demisto/content/blob/0805e4a58b64f65d15be59cbd28b2f4f8bd8ae80/Packs/CommonScripts/Scripts/GetIndicatorDBotScore/GetIndicatorDBotScore_test.py#L39-#L63